src/transformers/modeling_tf_bert.py

```diff
@@ -275,7 +275,10 @@ def call(self, inputs, training=False):
             context_layer, (batch_size, -1, self.all_head_size)
         )  # (batch_size, seq_len_q, all_head_size)
 
-        outputs = (context_layer, attention_probs) if self.output_attentions else (context_layer,)
+        outputs = (
+            (context_layer, attention_probs) if cast_bool_to_primitive(output_attentions) is True else (context_layer,)
```

Comment:
@thomwolf @LysandreJik we need to add a `cast_bool_to_primitive` function in all tf files to deal with keras serialization and compilation